### PR TITLE
Updated Examples for DNS and IPAM objects

### PR DESCRIPTION
**Pull request by unasra**

### CODE REVIEW

File: docs/resources/dns_view.md [hunk 1]
```diff
@@ -33,7 +33,50 @@ resource "bloxone_dns_view" "example" {
...
+      algorithm  = 8
+      public_key = "AwEAAejpWrcCPGWEoiebhWKSdT6LcMGBsoXadKu1XNthMZUvx3P92HNE4J3q3EtAX8pnTsNShrsDvvgn4hmCsrURMLx/g+76JtLU5pdbtrGFjelHAuMrzLgFzpuA5Ct9THth5Hto6c0rl4yzz3qT3+I/rnUYrL/zd9zKWyMp1A9KlHqwCA3JbFZfl4IKBD2/g+GScEcpnDfUUVDU+7qRZkZ4BhBQ4a6Em73zggz/crcDtwc1cHcRP0DGbekZhF29+yjTPW4zKqGUHW8ZtP49ZMXOTY42epeiddFNy0Ze2jbTg99CnKvAxIKzYInUaPJ04rgMyeuVWpRKsVetJemhCaj9lEs="
+      zone       = "example.com."
+      sep        = true
+    }
+  ]
+  zone_authority = {
+    default_ttl       = 28800
+    expire            = 2419200
+    mname             = "ns.b1ddi.example.com"
```
Comment (venkat-iblox) [inline docs/resources/dns_view.md:73]:
```suggestion
    mname             = "ns.example.com"
```

File: docs/resources/ipam_address_block.md [hunk 2]
```diff
@@ -38,6 +38,36 @@ resource "bloxone_ipam_address_block" "example_tags" {
...
+      {
+        type  = "client_text"
+        value = "001d.a18b.36d0"
+      },
+      {
+        type  = "client_hex"
+        value = "333964392D4769302F31"
+      }
+    ]
+
```
Comment (venkat-iblox) [inline docs/resources/ipam_address_block.md:69]: nit
```suggestion
```

File: examples/resources/bloxone_dns_auth_zone/resource.tf [hunk 3]
```diff
@@ -7,4 +7,38 @@ resource "bloxone_dns_auth_zone" "example" {
...
+      access  = "allow"
+      element = "ip"
+      address = "192.168.1.1"
+    },
+    {
+      access  = "deny"
+      element = "any"
+    }
+  ]
+
```
Comment (venkat-iblox) [inline examples/resources/bloxone_dns_auth_zone/resource.tf:43]: nit
```suggestion
```

File: examples/resources/bloxone_dns_view/resource.tf [hunk 4]
```diff
@@ -19,6 +19,49 @@ resource "bloxone_dns_view" "example" {
...
+      algorithm  = 8
+      public_key = "AwEAAejpWrcCPGWEoiebhWKSdT6LcMGBsoXadKu1XNthMZUvx3P92HNE4J3q3EtAX8pnTsNShrsDvvgn4hmCsrURMLx/g+76JtLU5pdbtrGFjelHAuMrzLgFzpuA5Ct9THth5Hto6c0rl4yzz3qT3+I/rnUYrL/zd9zKWyMp1A9KlHqwCA3JbFZfl4IKBD2/g+GScEcpnDfUUVDU+7qRZkZ4BhBQ4a6Em73zggz/crcDtwc1cHcRP0DGbekZhF29+yjTPW4zKqGUHW8ZtP49ZMXOTY42epeiddFNy0Ze2jbTg99CnKvAxIKzYInUaPJ04rgMyeuVWpRKsVetJemhCaj9lEs="
+      zone       = "example.com."
+      sep        = true
+    }
+  ]
+  zone_authority = {
+    default_ttl       = 28800
+    expire            = 2419200
+    mname             = "ns.b1ddi.example.com"
```
Comment (venkat-iblox) [inline examples/resources/bloxone_dns_view/resource.tf:59]:
```suggestion
    mname             = "ns.example.com"
```

File: examples/resources/bloxone_ipam_address_block/resource.tf [hunk 5]
```diff
@@ -23,4 +23,34 @@ resource "bloxone_ipam_address_block" "example_tags" {
...
+      {
+        type  = "client_text"
+        value = "001d.a18b.36d0"
+      },
+      {
+        type  = "client_hex"
+        value = "333964392D4769302F31"
+      }
+    ]
+
```
Comment (venkat-iblox) [inline examples/resources/bloxone_ipam_address_block/resource.tf:54]: nit
```suggestion
```

File: docs/resources/dns_auth_zone.md [hunk 6]
```diff
@@ -22,6 +22,40 @@ resource "bloxone_dns_auth_zone" "example" {
...
+      access  = "allow"
+      element = "ip"
+      address = "192.168.1.1"
+    },
+    {
+      access  = "deny"
+      element = "any"
+    }
+  ]
+
```
Comment (venkat-iblox) [inline docs/resources/dns_auth_zone.md:58]: nit
```suggestion
```

File: docs/resources/dns_view.md (hunk 1 above)
Comment (unasra) [reply docs/resources/dns_view.md]: Fixed

File: docs/resources/ipam_address_block.md (hunk 2 above)
Comment (unasra) [reply docs/resources/ipam_address_block.md:69]: Reviewed

File: examples/resources/bloxone_dns_auth_zone/resource.tf (hunk 3 above)
Comment (unasra) [reply examples/resources/bloxone_dns_auth_zone/resource.tf:43]: Resolved

File: examples/resources/bloxone_dns_view/resource.tf (hunk 4 above)
Comment (unasra) [reply examples/resources/bloxone_dns_view/resource.tf]: addressed

File: examples/resources/bloxone_ipam_address_block/resource.tf (hunk 5 above)
Comment (unasra) [reply examples/resources/bloxone_ipam_address_block/resource.tf:54]: Taken Care of !

File: docs/resources/dns_auth_zone.md (hunk 6 above)
Comment (unasra) [reply docs/resources/dns_auth_zone.md:58]: Resolved

**Comment by venkat-iblox** [comment]
Changes LGTM. I'll let @mathewab finish up the review

**Pull request merged**